### PR TITLE
Add gcc builds for armv7hf.

### PR DESCRIPTION
**Pull request by mbeach**
I need these builds for a project I'm working on, and thought they might be generally useful.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,6 +18,18 @@ matrix:
         env: CONAN_GCC_VERSIONS=6 CONAN_DOCKER_IMAGE=lasote/conangcc6
       - <<: *linux
         env: CONAN_GCC_VERSIONS=7 CONAN_DOCKER_IMAGE=lasote/conangcc7
+      - <<: *linux
+        env: CONAN_GCC_VERSIONS=8 CONAN_DOCKER_IMAGE=lasote/conangcc8
+      - <<: *linux
+        env: CONAN_GCC_VERSIONS=4.9 CONAN_DOCKER_IMAGE=lasote/conangcc49 CONAN_ARCHS=armv7hf
```
Comment (Croydon) [inline .travis.yml]: Why aren't you using the specific armv7hf containers?

Also please use the new `conanio/gcc49` (and so one) containers. While you at it, could you please update all container names?

**Review by Croydon — CHANGES_REQUESTED** [review]
Please update the container names

File: .travis.yml (hunk 1 above)
Comment (mbeach) [reply .travis.yml]: Well, I am using the armv7hf specific containers, in that that's what Conan Package Tools ends up using, apparently based on the setting for CONAN_ARCHS; please see lines 812-849 of https://travis-ci.com/mbeach/conan-zstd/jobs/155308489. I seem to recall that when I first tried building for armv7hf I specified the armv7hf container explicitly AND set CONAN_ARCHS, and it tried to use conangcc7-armv7hf-armv7hf. I suppose I could have tried to leave CONAN_ARCHS unset, but was not sure if that would have some unintended negative consequences.

Will change the container names though.

**Comment by mbeach** [comment]
Hi @Croydon, I have updated all of the container names.

File: .travis.yml (hunk 1 above)
Comment (Croydon) [reply .travis.yml]: Good to know. Thanks

**Review by Croydon — APPROVED** [review]
LGTM

**Review by uilianries — APPROVED** [review]
LGTM

**Pull request merged**